lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -391,21 +481,46 @@ sub get_test_results {
     return $result;
 }
 
+$json_schemas{get_test_history} = joi->object->strict->props(
+    offset => joi->integer->min(0),
+    limit => joi->integer->min(0),
+    frontend_params => joi->object->strict->props(
+        domain => $zm_validator->domain_name->required,
+        ipv4 => joi->boolean,
+        ipv6 => joi->boolean,
+        ds_info => joi->array->strict->items(
+            $zm_validator->ds_info
+        ),
+        nameservers => joi->array->strict->items(
+            $zm_validator->nameserver
```

Comment:
This is inconsistent with the API documentation. Could you fix the one that's wrong?

Reply:
Done